db.py

```diff
@@ -53,10 +53,10 @@ class Course(db.Model):
     __tablename__ = 'course'
     id = db.Column(db.Integer, primary_key=True, autoincrement=True)
     year = db.Column(db.Integer, primary_key=True)
-    code = db.Column(db.String)
-    title = db.Column(db.String)
+    code = db.Column(db.String(10))
+    title = db.Column(db.String(50))
```

Comment:
```suggestion
    title = db.Column(db.String(100))
```

This may grow very quickly...